### ISSUE TITLE
Updates necessary for newly added test cases

### ISSUE THREAD

**Issue by dzhulgakov** [onnx/onnx-coreml#6]
We're shooting for ONNX v1 announcement at NIPS next week and thus it'd be awesome to make sure that all tests are passing with the latest ONNX master.

We added a few backend test cases upstream in ONNX (by exporting some of the PyTorch's tests) and some of them are not passing the tests in CoreML.

Notable issues:
1) #5 addresses padding and tries to fix some reshapes
2) Some of the ops are not properly mapped (e.g. PRelu)
3) Some of the ops variants are not supported in CoreML (e.g. arbitrary dimensional SoftMax)

For 3) we need to figure out some way (throwing special exception?) to mark the tests as "expected unsupported behavior" instead of just failing them blindly.

Help on looking into these is appreciated.

**Comment by opedge** [comment]
For 2) I'm updating converter to support ops.
As for 3) I think the special "Unsupported" exception will be ok, will the onnx backend.text support this?

**Comment by opedge** [comment]
I've updated the converter just now to support Elu, PRelu, Tanh, Exp, Flatten, Max, Softsign, Softplus, Neg, Split, Log operators.
Also I had to exclude a lot of new tests that they are not compatible with CoreML:
- Unsupported input shape: test_PReLU, test_Conv3d, test_BatchNorm3d, test_BatchNorm1d, test_AvgPool2d, test_AvgPool3d, test_GLU, test_Linear, test_LogSoftmax, test_MaxPool1d, test_MaxPool2d, test_MaxPool3d
- Unsupported attributes or operators: test_clip (Clip op), test_softmax_lastdim, test_softmax_functional_dim3, test_log_softmax_dim3, test_log_softmax_lastdim, test_ZeroPad2d_negative_dims, test_Softsign (uses Add op with broadcasting), test_Softmax, test_Softmin, test_Embedding (Gather op).

**Comment by bhushan23** [comment]
tests are enabled under onnx backend node test.
closing
